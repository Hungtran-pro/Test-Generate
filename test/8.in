32
1964
1609
1651
1932
1759
1846
1793
1852
1363
1785
1863
1803
1984
1348
1060
1525
1413
1503
1248
1685
1399
1239
1636
1199
1409
1826
1812
1098
1903
1535
1784
1751
