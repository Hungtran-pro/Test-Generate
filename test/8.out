179362744
871504337
57868746
688281960
985899922
890821672
929320871
783243895
578554941
106154329
447329019
918750187
869904827
783311150
152201777
220046716
712930857
749735674
160115778
326794857
271512854
853929352
307316661
593367349
388116114
513878197
351622543
244661400
329107004
89324706
1199448
967201948
